container_pipeline/workers/base.py

```diff
@@ -0,0 +1,62 @@
+import json
+import logging
+import os
+
+from container_pipeline.lib import settings
+from container_pipeline.lib.openshift import Openshift
+from container_pipeline.lib.queue import JobQueue
+
+
+class BaseWorker(object):
+    """Base class for pipeline workers"""
+
+    def __init__(self, logger=None):
+        self.queue = JobQueue(host=settings.BEANSTALKD_HOST,
+                              port=settings.BEANSTALKD_PORT)
+        self.logger = logger or logging.getLogger('console')
+        self.openshift = Openshift(logger=self.logger)
+        self.openshift.login()
+
+    def handle_job(self, job):
+        """Handle job"""
+        raise NotImplementedError
```

Comment:
We don't have an implementation of this exception yet in the refactor. Just FYI.